spec/14-classes.md

```diff
@@ -387,6 +387,14 @@ Widget::__callStatic('sMethod', array(NULL, 1.234))
 <pre>
   <i>const-declaration:</i>
     const  <i>name</i>  =  <i>constant-expression</i>   ;
+
+  <i>class-const-declaration:</i>
```

Comment:
You should also update the references to const-declaration in class-member-declaration and maybe other places.